client/src/no-match/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+
+import { Titlebar } from "../ui/molecules/titlebar";
+
+import "./index.scss";
+
+export function NoMatch() {
```

Comment:
Rename this (and its file) to "PageNotFound" or something?

Reply:
If that is its sole purpose, then yes by all means rename to `PageNotFound`

Reply:
Done.